### DISCUSSION TITLE
Request for Comments: Account Creation Date Visibility

### DISCUSSION THREAD

**Discussion by jmking-iftas** [mastodon/mastodon#32874] [General]
I am seeking comment on the pros and cons of having the account creation date visible on public views of an account profile. 

I believe there is value to exploring if having the account creation date visible publicly in the interface is overall a net negative for the network. 

This discussion assumes that for purposes of moderation and discoverability the creation date *should* be taken into account, and is only discussing the public visibility of the creation date on the member profile when viewed by an authenticated or unauthenticated client in a public web or client view.

![A member profile of a fictional member named Generic User 1, showing the Mastodon web client version of an account profile, including display name, username, biography, account creation date, web site link, and total posts, followings and followers for this account.](https://github.com/user-attachments/assets/6cb7bfa7-87be-4c35-bb35-956647976068)

## Possible Pros

- A visible account creation date may enhance credibility, especially if the account has been active for some time. This can help distinguish between longtime community members, and newer accounts. 
- If multiple accounts for a given member/publication/organisation exist, knowing which the account creation date may improve perception of authenticity.
- If a member is concerned about reporting e.g. spam or malicious bots, knowing the account creation date may help that member in their decision to report an account.
- New accounts may be a signal that the member is seeking to engage on the network possibly inspiring other community members to welcome the newer member.

## Possible Cons

- Displaying the account creation date may erode privacy for a member who may not want their account age known. Members of vulnerable communities may fear being judged or targetted based on their account's perceived inexperience. 
- Having the creation date visible may inspire gatekeeping, where members may feel they have some level of enhanced credibility or authority over newer members.
- If older accounts are perceived as being more credible generally, this disadvantages and discourages engagement from new members who can see that credibility on display.
- Account creation dates may lead to third party assumptions about the member's expertise, trustworthiness, or allow for other negative stereotyping based purely on account age (old /or/ new).

## Additional Thoughts

 - We have rel=me verification, number of followers, number of posts, these are all visible components of a given account's history and credibility.
 - Account age is far less useful than account recency, or account post history. A large number of accounts were created in Q4 2022 and abandoned shortly thereafter. Does showing the account creation date help or hinder public perception of these accounts?
 - Account age does not equal account engagement. (Nor does post count.)
 - Account age does not carry over from an account move / migration. It is not a true reflection of many members' actual longevity on the network.

## Suggested Approaches

1. Remove creation date from public interface views.
2. Make it a member-configurable toggle to display or not display the creation date in the interface. Or, allow members to make it visible to followers-only or similar privacy settings that restrict the info to trusted members only.
3. Remove the year, but show the creation month and date. This allows for emergent behaviour around the creation month or day, but removes longevity from the public sphere.
4. Add a "New" sticker for accounts that less than $days old (3? 7? 28?) - this can help in identifying new members who may want to be welcomed or followed, or to identify newly-created spam accounts  (although I am aware of accounts created and held for some time before being used for spamming/unsolicited direct mentions).
5. (If 4, if we add a "new" flag/sticker) Make this an admin-level setting so that communities that encourage mentorship can do so, and communities that want to avoid any potential stigmatisation due to newness or perceived newness can disable this sticker.
6. If we want to signal long-term engagement (and I use the term engagement very loosely here), add milestones such as "Joined this year", "Joined over a year ago" or "Joined over five years ago" or similar.

## Questions for Comment

 - Why do we show account creation date in the interface? Knowing the reason for the date being visible in the first place will help this conversation.
 - Is there any published research on account creation date visibility (I couldn't find any...)
 - Should it be an option for the member to choose whether or not their account creation date is visible in the interface?
 - Should the account creation date be removed at the interface level, never showing it to public viewers?
 - - If that were to happen, should it be made available when being direct mentioned?
 - Do you personally want your account creation date visible or not visible to the public in the interface. Why, or why not?

**Comment by nikclayton** [comment]
Clients can use the account creation date to implement anti-harassment features, like "Hide replies from accounts < 28d old" (or notifications, or similar).

This can help people who are facing targeted harassment from people creating new sock puppet accounts.

**Reply by jmking-iftas** [reply]
If there was a "new account" flag that minimised or hid the actual date, but still had a signal for "new" (for varying definitions of new) would that meet the need? Or do you believe individual members should choose the number of days? See (4) above.

**Reply by nikclayton** [reply]
The latter. A "new account" flag leaves the client/user dependent on the server's configuration for this, and complicates the client's UI.

It also prevents users from making more nuanced choices, e.g., where notifications are hidden from accounts < X days old, DMs < Y days old, where X and Y are different numbers.
